Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -98,6 +100,40 @@ def homepage(homepage = nil)
       @homepage ||= homepage
     end
 
+    def language(*args, default: false, &block)
+      if !args.empty? && block_given?
+        @language_blocks ||= {}
+        @language_blocks[args] = block
+
+        return unless default
+
+        unless @language_blocks.default.nil?
+          raise CaskInvalidError.new(token, "Only one default language may be defined")
+        end
+
+        @language_blocks.default = block
+      else
+        language_eval
+        @language
+      end
+    end
+
+    def language_eval
+      return if instance_variable_defined?(:@language)
+
+      return unless instance_variable_defined?(:@language_blocks)
+
+      MacOS.languages.map(&Locale.method(:parse)).any? { |locale|
```

Comment:
Seems like `each` is a better fit here than `any?` since you're not using the return value, as a boolean or otherwise.